drivers/hmis/spec/support/ac_hmis_loader_helpers.rb

```diff
@@ -4,9 +4,13 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
-# frozen_string_literal: true
-
 module AcHmisLoaderHelpers
+  def self.included(base)
```

Comment:
disable broken tests. Should be deleted in the near future but that probably should happen in another branch